### PR TITLE
Preview

### PR DESCRIPTION
**Pull request by andyofsmeg**

### CODE REVIEW

File: content/blog/2020-07-17-risk-assessment-application.Rmd [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+Towards the end of 2019, the R Validation Hub received an adidtional grant from the R Consortium to progress the next phase of our roadmap and produce a risk assessment app to compliment the [riskmetric package](/blog/2020/06/09/2020-06-02-riskmetric-intro-jun-2020/).  In early 2020, [Fission Labs](https://www.fissionlabs.com/) were selected as our partner to build the first iteration of the application.
+
+## Workflow
+
+### Roles
+
+This first version of the app implements a very simple workflow that can be tailored to an organisation's needs.  There is no log-in, the app detects the user's ID and asks them for their role.  
+
+![](/blog/2020-07-17-risk-assessment-application_files/roles.png){width=820px}
```
Comment (andyofsmeg) [inline content/blog/2020-07-17-risk-assessment-application.Rmd]:
```suggestion
![](/blog/2020-08-05-risk-assessment-application_files/roles.png){width=820px}
```

File: content/blog/2020-08-05-risk-assessment-application.Rmd [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Once packages have been uploaded to the DB, the main panel on the left can be used to select the package (and version) in order to review its metrics.  The metrics are split over three tabs:
+
+* Maintenance Metrics
+* Community Usage Metrics 
+* Testing Metrics
+
+Each individual tab provides a mechanism for commenting on the metrics.  Comments within the sections are design to be conversational, in other words a reviewer can make multiple comments alongside other reviewers.  The reviewer's ID, role and a date-time stamp are recorded along with the comment.
+
+![](/blog/2020-07-17-risk-assessment-application_files/risk_assess.png){width=820px}
```
Comment (andyofsmeg) [inline content/blog/2020-08-05-risk-assessment-application.Rmd:62]:
```suggestion
![](/blog/2020-08-05-risk-assessment-application_files/risk_assess.png){width=820px}
```

**Comment by marlycormar** [comment]
Hi @andyofsmeg, it looks great overall. I fixed minor typos/wording issues and added them here: https://github.com/marlycormar/pharmaR.github.io/commit/7e4be2cbca464339470eea5d9749e3caeae45b5c (I don't have pull rights). Feel free to merge them.

Also, some of the images are not displaying on the blog.

File: content/blog/2020-08-05-risk-assessment-application.Rmd [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+> *Fission Labs is a software product development services company delivering product life-cycle management and high-end scalable technology solutions. Our mission is to deliver complex products through simple and efficient engineering.*
+
+## So what does the app do?
+
+The aim of the app is to provide an interactive user interface to the riskmetric package, with metrics categorised into 'maintenance', 'community usage' and 'testing'.  The review team can then use the app to record their comments on the metrics.  Both the metrics and the user comments are stored in an underlying database.  This makes it possible to stop and start reviews at the reviewer's convenience.
+
+Once an a review is complete, a final summary comment on the package can be provided before a final decision is made on the package.  In line with our [white paper](/white-paper/), the *decision* is an overall risk score: low; medium; high.  At this point (or indeed at any point) any of the users could then generate either an HTML or DOCX package report containing the metrics, comments and some additional high level information about the package.
```
Comment (marlycormar) [inline content/blog/2020-08-05-risk-assessment-application.Rmd:26]:
```suggestion
Once a review is complete, a final summary comment on the package can be provided before a final decision is made on the package.  In line with our [white paper](/white-paper/), the *decision* is an overall risk score: low; medium; high.  At this point (or indeed at any point) any of the users could then generate either an HTML or a DOCX package report containing the metrics, comments and some additional high level information about the package.
```

File: content/blog/2020-08-05-risk-assessment-application.Rmd [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+For those looking for a littlemore detail, I've provided a little more detail in the following sections...
```
Comment (marlycormar) [inline content/blog/2020-08-05-risk-assessment-application.Rmd:28]:
```suggestion
For those looking for more information on the app, I've provided further details in the following sections...
```

File: content/blog/2020-08-05-risk-assessment-application.Rmd [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+### A Package assessment
+
+Once packages have been uploaded to the DB, the main panel on the left can be used to select the package (and version) in order to review its metrics.  The metrics are split over three tabs:
+
+* Maintenance Metrics
+* Community Usage Metrics 
+* Testing Metrics
+
+Each individual tab provides a mechanism for commenting on the metrics.  Comments within the sections are design to be conversational, in other words a reviewer can make multiple comments alongside other reviewers.  The reviewer's ID, role and a date-time stamp are recorded along with the comment.
```
Comment (marlycormar) [inline content/blog/2020-08-05-risk-assessment-application.Rmd:60]:
```suggestion
Each individual tab provides a mechanism for commenting on the metrics.  Comments within the sections are designed to be conversational, in other words a reviewer can make multiple comments alongside other reviewers.  The reviewer's ID, role and a date-time stamp are recorded along with the comment.
```

File: content/blog/2020-08-05-risk-assessment-application.Rmd [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+![](/blog/2020-08-05-risk-assessment-application_files/risk_assess.png){width=820px}
```
Comment (marlycormar) [inline content/blog/2020-08-05-risk-assessment-application.Rmd:62]: This image doesn't show on the preview site you shared, but it does when I create the blogdown locally. Maybe re-generating the html blog post will fix this.

**Comment by joseph-rickert** [comment]
in first sentence: should be "complement".

File: content/blog/2020-08-05-risk-assessment-application.Rmd (hunk 6 above)
Comment (andyofsmeg) [reply content/blog/2020-08-05-risk-assessment-application.Rmd:62]: I meant to mention this - I noticed this after looking at the preview and updated the source.  As you say, I need to re-generate.

**Pull request merged**